lib/public_key/test/public_key_SUITE.erl

```diff
@@ -1132,8 +1132,11 @@ pkix_verify_hostname_options(Config) ->
     true = public_key:pkix_verify_hostname(Cert, [{dns_id,"abb.bar.example.com"}]),
     false = public_key:pkix_verify_hostname(Cert, [{dns_id,"example.com"},
                                                    {dns_id,"abb.bar.example.com"}],
-                                            [{fqdn_fun,fun(_)->undefined end}]).
-    
+                                            [{fqdn_fun,fun(_)->undefined end}]),
+    {ok, PrefixBin} = file:read_file(filename:join(DataDir,"prefix-dots.pem")),
```

Comment:
can you add a code comment explaining what is specific about prefix-dots.pem  and relevant for test lines below?
I guess maybe this: "Subject: CN = ..a"

Reply:
Yes subject "CN=..a" and the whole name should match.  ".a" should not match "..a" which it wrongly did!

Reply:
Add a comment